Bio/Phylo/PAML/baseml.py

```diff
@@ -175,7 +179,11 @@ def run(self, ctl_file=None, verbose=False, command="baseml",
 
 
 def read(results_file):
-    """Parse a BASEML results file."""
+    """Parse a BASEML results file.
+
+    Check that the file exits, that the results file is not empty, and then
+    parse the file. Check there is a version in the results and then return
+    the results. Otherwise raise an exception."""
```

Comment:
Move the triple quote to its own line please - this and three other cases are failing the automated style checks using flake8:

```
$ flake8 Bio/
Bio/Phylo/PAML/baseml.py:93:1: D209 Multi-line docstring closing quotes should be on a separate line
Bio/Phylo/PAML/baseml.py:181:1: D209 Multi-line docstring closing quotes should be on a separate line
Bio/Phylo/PAML/codeml.py:94:1: D209 Multi-line docstring closing quotes should be on a separate line
Bio/Phylo/PAML/codeml.py:193:1: D209 Multi-line docstring closing quotes should be on a separate line
```